lib/rules/lint-eol-last.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const Rule = require('./base');
+
+module.exports = class EolLast extends Rule {
+  visitor() {
+    return {
+      Program: {
+        exit(node) {
+          let line = node.loc.start.line;
+          let column = node.loc.start.column;
+
+          // yielded content makes it in here too
+          //
+          // `{{#my-component}}
+          //   test
+          // {{/my-component}}`
+          //
+          // becomes
+          //
+          // `
+          //   test
+          // `
+          //
+          // check for that case
+          if (line !== 1 || column !== 0) {
```

Comment:
I'd rather keep a closure scope variable that says we have processed and noop.

Basically like https://github.com/rwjblue/ember-template-lint/blob/master/lib/rules/lint-template-length.js#L77.

Reply:
I tried that but it won't work. The inner content goes through the `exit` first, so it's the reverse order of what I need.